autoload/clap.vim

```diff
@@ -309,6 +309,11 @@ endif
 function! s:parse_opts(args) abort
   let idx = 0
   for arg in a:args
+    if arg ==# '--'
+      let g:clap.context.query = join(a:args[idx+1 :], ' ')
+      let idx = len(a:args)
+      break
+    endif
```

Comment:
Nice to have the docs updated since this patch changes how `query` is parsed and the way of users use it.